### DISCUSSION TITLE
1.1.0 版本社区意见征集 / Community Input for Version 1.1.0 Features

### DISCUSSION THREAD

**Discussion by suikodev** [AstraSurge/gpteams#27] [Maintainer-Initiated Feature Discussions]
GPTeams 是一个由社区驱动的项目，大部分功能都会征集社区的意见与建议。目前 1.0.0 版本的开发已经进入尾声了，预计将于下周一（2023-4-3）前经过发布稳定版。目前的规划是会在每两周的周末发布新的稳定版本。

现在征集关于项目 1.1.0 版本的社区意见。请分享你对于希望在此次更新中实现的功能或改进的想法、建议和反馈。你的意见会塑造项目的未来发展方向。

如果你赞同其他人提出的意见和建议，请为他们点赞，这将帮助我们了解各个建议的受欢迎程度。

当前已经决定在 1.1.0 实现的新功能有：

1. 用户可选择将本地某个会话同步至云端
2. 用户可将会话分享给团队中的其他成员
3. 用户分组。分组后可以设置不同组的对话使用限制，GPT 模型等
4. [chat with PDF](https://www.chatpdf.com/)

感谢 @Pinkpiglet 在 1.0.0 版本中的积极反馈。

GPTeams is a community-driven project that seeks input and suggestions from its users for most of its features. The development of version 1.0.0 is now in its final stages and is expected to be released as a stable version by next Monday (April 3, 2023). The current plan is to release a new stable version every two weeks on weekends.

We are currently seeking feedback and suggestions from the community for version 1.1.0 of the project. Please share your ideas, suggestions, and feedback on the features or improvements you would like to see in this update. Your input will help shape the future direction of the project.

If you agree with other people's suggestions and ideas, please give them a thumbs up. This will help us understand the popularity of each proposal.

Currently, the following new features have been decided to be implemented in version 1.1.0:

1. Users can choose to synchronize a local session with the cloud.
2. Users can share sessions with other members of the team.
3. Users can create groups and set usage restrictions for different groups, such as GPT models.
4. [Chatting with PDF](https://www.chatpdf.com/)

Finally, we would like to thank @Pinkpiglet for their positive feedback on version 1.0.0.

**Comment by First-Power** [comment]
是否具有防止多人登录同一账户功能

**Reply by zzkk74006** [reply]
您的意见我们已在讨论，该功能的进展会在这里通知您，也请您持续关注我们的更新，最后十分感谢您宝贵的意见！

**Reply by suikodev** [reply]
@First-Power 防止多人登录同一账户是你的目的吗？如果目的只是想防止用户分享账户导致滥用的情况，完全是可以考虑对用户进行限流的方案的。1.0.0 会做系统级别的用户限流功能，在 1.1.0 增加角色分组后就可以对用户组进行限流了

**Comment by LuckyWang6** [comment]
可否支持mongodb数据库迁移，因为之前也做了一个项目用的是原项目的PR

https://github.com/Chanzhaoyu/chatgpt-web/pull/589

**Reply by suikodev** [reply]
我们可以做一个白名单导入的功能，不知道能否满足你的需求

**Comment by jurieo** [comment]
1、支持邀请码机制，如果关闭了注册，使用邀请码可以注册登录。
2、兑换码支持。后台支持生成兑换码，兑换码兑换聊天时间或次数。
3、支持注册用户限制免费聊天时间或固定次数，超出需付费或者购买兑换码。
开发人员可否考虑以上需求，谢谢啦

**Reply by suikodev** [reply]
我们期望的 GPTeams 的核心价值是帮助组织管理者轻松引入 ChatGPT 并通过它提升组织成员的工作效率。因此，我们的开发主线会是基于 ChatGPT 来实现更多实用的功能。我们会讨论你提出的这些需求，但不一定所有需求都会被全部实现。之后的具体进展会同步到这里

**Reply by jurieo** [reply]
嗯嗯，非常感谢。有计划提供docker-compose的部署方式吗？因为看到docker-compose的文件夹里还是原repo的文件。

**Reply by suikodev** [reply]
会的
下周一发布 1.0.0 稳定版本前会出一个详细的wiki, 包含 docker, docker-compose, node.js 的详细部署方式

**Comment by hongminy** [comment]
请问作者可否加入每个用户账户token独立计算的功能, 方便限制每个用户token的使用量

**Reply by suikodev** [reply]
我觉得可以，而且比根据请求次数来计算是更合理的，但是我们这边现在的开发资源不太够，你这个估计得 1.2.0 版本才会做。
